poetry/parts/header.html

```diff
@@ -1 +1,25 @@
-<!-- wp:pattern {"slug":"poetry/header"} /-->
\ No newline at end of file
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"32px","bottom":"64px"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group" style="padding-top:32px;padding-bottom:64px"><!-- wp:group {"style":{"spacing":{"blockGap":"64px"}},"layout":{"type":"flex","flexWrap":"nowrap"}} -->
+<div class="wp-block-group"><!-- wp:group {"style":{"spacing":{"blockGap":"88px"}},"layout":{"type":"flex","orientation":"vertical"}} -->
+<div class="wp-block-group"><!-- wp:site-title {"level":0,"style":{"typography":{"fontStyle":"normal","fontWeight":"600"}},"fontSize":"medium"} /-->
+
+<!-- wp:group {"style":{"spacing":{"blockGap":"48px"}},"layout":{"type":"flex","orientation":"vertical"}} -->
+<div class="wp-block-group"><!-- wp:paragraph {"style":{"typography":{"lineHeight":"0.8"}},"fontSize":"xx-large","fontFamily":"rozha-one"} -->
+<p class="has-rozha-one-font-family has-xx-large-font-size" style="line-height:0.8">Food <br>for thoughts</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph {"style":{"typography":{"lineHeight":"1.5"}},"fontSize":"medium"} -->
+<p class="has-medium-font-size" style="line-height:1.5">A collection of valuable thoughts by a very egocentric person.</p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:group --></div>
+<!-- /wp:group -->
+
+<!-- wp:image {"width":"320px","height":"379px","sizeSlug":"full","linkDestination":"none"} -->
+<figure class="wp-block-image size-full is-resized"><img src="http://community-themes.local/wp-content/themes/poetry/assets/images/feather.png" alt="" class="" style="width:320px;height:379px"/></figure>
+<!-- /wp:image --></div>
+<!-- /wp:group --></div>
+<!-- /wp:group -->
+
+<!-- wp:spacer {"height":"50px"} -->
+<div style="height:50px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
```

Comment:
we still need the contents of the header to live inside a pattern so we can make the URLs dynamic using PHP functions and translate all the strings

Reply:
@MaggieCabrera Thanks for your review. Just fixed the header and index templates to call the respective patterns, as it was originally. And added the links to site title and post title in the respective patterns. Let me know if there is anything else I should change.

I'm still getting "PR run failed: Preview Theme Changes". Not sure if there is something I can do on my end to fix this.

Reply:
No, it's a problem with permissions when the person creating the PR doesn't have enough permissions in the repo. It might be related to the PR being in a fork. I'm looking into it, but it doesn't block your PR